ui/webui/test/check-storage

```diff
@@ -48,5 +48,31 @@ class TestStorage(MachineCase):
         b.wait_js_cond('window.location.hash === "#/installation-destination"')
         b.wait_visible("#installation-destination.pf-m-current")
 
+# We can't run this test case on an existing machine,
+# with --machine because MachineCase is not aware of add_disk method
+class TestStorageExtraDisks(MachineCase):
+    MachineCase.machine_class = VirtInstallMachine
+
+    def testLocalDisksSyncNew(self):
+        b = self.browser
+        m = self.machine
+
+        m.add_disk(5)
```

Comment:
Is this the right place to add the disk? I would expect it to be added between `b.wait_not_present("#vdb")` and `b.click("#rescan-disks")`.

Reply:
The point is that even if you add the disk  before loading the page, since initialization of modules is done, it will not appear in the UI, until we click rescan.

So the current command order is testing exactly what I wanted.

I can add the wait_not_present("#vdb") before the add_disk as well, but that' expected to not be there.

Reply:
Oh, I see. Maybe add that, or just a comment? It's not clear we are testing that the other disk is missing, too.

Reply:
Added comment. Note, that actually it does not make any sense to call wait_not_present(#vdb) anywhere before i.next(), because only at that point, we are at the storage step in the wizard, which shows the disks.